client/app/shared/modals/model-delete/model-delete.component.ts

```diff
@@ -1,8 +1,11 @@
-import { Component } from '@angular/core'
+import { Component, ViewChild } from '@angular/core'
+
 import { Deletable, DeletableModalService } from '../../models/interfaces/modal-model'
 import { Notifications } from '../../services/notifications.service'
 import { DeleteService } from '../../services/delete-service.service'
 
+declare var jQuery: any
```

Comment:
`jQuery` is not part of the package.json

Reply:
You're right. It's part of the Semantic pacakge. Thereafore, it only needs to be declared.